### PR TITLE
feat(TD-13912): support schemaless insert

### PR DESCRIPTION
**Pull request by zitsen**

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM

File: Cargo.toml [hunk 1]
```diff
@@ -42,7 +42,8 @@ proc-test-catalog = { path = "./proc-test-catalog", version = "0.1.0" }
 test-catalog = { path = "./test-catalog", version = "0.1.0" }
 
 [features]
-default = ["stmt", "cleanup"]
+default = ["stmt", "cleanup", "sml"]
```
[the remark this answers is not in the document]
Comment (zitsen) [reply Cargo.toml:45]: Agree, will change.

File: README.md [hunk 2]
```diff
@@ -15,13 +15,14 @@ if you use the default features, it'll depend on:
 - [TDengine] Client library and headers.
 - clang because bindgen will requires the clang AST library.
 
-## Fetures
```
[the remark this answers is not in the document]
Comment (zitsen) [reply README.md:18]: Thanks, will change.

[the remark this answers is not in the document]
Comment (zitsen) [reply README.md:18]: Done

**Review by dingbo8128 — APPROVED** [review]
LGTM

**Pull request merged**